src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,23 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    private static SessionFactory instance = initSessionFactory();
+
+    private HibernateUtil() {
+    }
```

Comment:
The `HibernateUtil` class should have a private default constructor to prevent instantiation. Good job following this practice as per the checklist.